shared/hybrid/AppDelegate+SalesforceHybridSDK.m

```diff
@@ -59,6 +59,17 @@ - (AppDelegate *)sfsdk_swizzled_init
     // Need to use SalesforceSDKManagerWithSmartStore when using smartstore
     [SalesforceSDKManager setInstanceClass:[SalesforceSDKManagerWithSmartStore class]];
     [SalesforceSDKManager sharedManager].appConfig = appConfig;
+    [SFAuthenticationManager sharedManager].advancedAuthConfiguration = SFOAuthAdvancedAuthConfigurationRequire;
```

Comment:
@bhariharan @khawkins Its supposed to be commented. Ill check it in. Remnants from going between testing and templates.